engine/buildkit/executor.go

```diff
@@ -43,10 +43,11 @@ import (
 )
 
 type ExecutionMetadata struct {
-	ClientID    string
-	SessionID   string
-	SecretToken string
-	Hostname    string
+	ClientID          string
+	SessionID         string
+	SecretToken       string
+	Hostname          string
+	SSHAuthSocketPath string
```

Comment:
nit: this shouldn't be in this top block - those are grouped together as the most important metadata parts.

It also should have a comment explaining what it's for.